packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -402,7 +404,7 @@ describe('Overhead controller', () => {
         })
       }
 
-      testInRequest(app, tests)
+      () => testInRequest(app, tests) // TODO: re-enable when fixed with tap
```

Comment:
A bare arrow function is a pretty weird way to comment something out. Can you just use a normal comment here?

Reply:
Sure, I was trying to avoid changing too many lines and came up with this, but I can comment out all the lines involved instead.

Reply:
Can't you just comment out this line?

Reply:
Fixed.